contracts/test/RFQ.t.sol

```diff
@@ -439,11 +439,11 @@ contract RFQTest is StrategySharedSetup {
 
     function testEmitSwappedEvent() public {
         RFQLibEIP712.Order memory order = DEFAULT_ORDER;
-        bytes memory makerSig = _signOrder(makerPrivateKey, order, SignatureValidator.SignatureType.EIP712);
-        bytes memory userSig = _signFill(userPrivateKey, order, SignatureValidator.SignatureType.EIP712);
-        bytes memory payload = _genFillPayload(order, makerSig, userSig);
+        bytes memory makerSig = _signOrder({ privateKey: makerPrivateKey, order: order, sigType: SignatureValidator.SignatureType.EIP712 });
+        bytes memory userSig = _signFill({ privateKey: userPrivateKey, order: order, sigType: SignatureValidator.SignatureType.EIP712 });
+        bytes memory payload = _genFillPayload({ order: order, makerSig: makerSig, userSig: userSig });
 
-        _expectEvent(order);
+        _expectEvent({ order: order });
```

Comment:
nit: This is also a single parameter function 😆

Reply:
OK ~